tests/gradients/test_spsa_gradient.py

```diff
@@ -30,17 +31,22 @@ def coordinate_sampler(indices, num_params, idx, seed=None):
     to the index ``indices[idx]``. This is a sequential coordinate sampler
     that allows to exactly reproduce derivatives, instead of using SPSA in the
     intended way."""
+    # pylint: disable=unused-argument
```

Comment:
Why do we need this?

Reply:
Ah! In PL the `# pylint: disable` message is usually written **above** the function (hence my confusion).

Reply:
Oh oops! Shall I move it to the line exactly above the `def ` statement? I wasn't aware sorry :D

Reply:
It's not a big deal 😄 

You can leave it like this